### PR TITLE
[datadog] Update agent to v5.11.2

### PR DESCRIPTION
**Pull request by johnaxel**
Updates the version of the Datadog Agent on Universe and enables service discovery by default for simpler monitoring of Docker containers.

### CODE REVIEW

**Comment by hkaj** [comment]
LGTM 👍

File: repo/packages/D/datadog/2/package.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+    "packagingVersion": "3.0",
+    "description": "Cloud-scale monitoring",
```
Comment (ryadav88) [inline repo/packages/D/datadog/2/package.json]: Can you add more details to the description

File: repo/packages/D/datadog/2/package.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        {
+            "name": "Simplified BSD license",
+            "url": "https://raw.githubusercontent.com/DataDog/dd-agent/master/LICENSE"
+        }
+    ],
+    "maintainer": "package@datadoghq.com",
+    "name": "datadog",
+    "postInstallNotes": "Datadog on DCOS successfully installed.\nHead over to https://app.datadoghq.com to monitor your DCOS cluster.",
+    "postUninstallNotes": "Datadog on DCOS successfully uninstalled.\n.",
+    "preInstallNotes": "Pass your API key as a package option (datadog.api_key).\nAlso pass the number of Mesos slaves as an option (datadog.instances).\nMarathon will make sure that the Datadog Agent runs only once per slave.",
```
Comment (ryadav88) [inline repo/packages/D/datadog/2/package.json]: Can you add a blurb about - where should users go to get the API key

**Comment by johnaxel** [comment]
Hi @ryadav88, I think I've addressed your feedback here, and have opened a PR with a getting-started doc for the package:

https://github.com/dcos/examples/pull/104

**Pull request merged**